graphql_client_cli/src/generate.rs

```diff
@@ -59,7 +59,7 @@ pub(crate) fn generate_code(params: CliCodegenParams) -> Result<(), failure::Err
         options.set_deprecation_strategy(deprecation_strategy);
     }
 
-    let gen = generate_module_token_stream(query_path.clone(), &schema_path, options)?;
+    let gen = generate_module_token_stream(query_path.clone(), &schema_path, options).map_err(|fail| fail.compat())?;
```

Comment:
Seems `rustfmt` wants this wrapped.

Reply:
Weird, I can't reproduce this locally. Merging for now.